nilearn/signal.py

```diff
@@ -641,3 +567,107 @@ def clean(signals, sessions=None, detrend=True, standardize='zscore',
                                detrend=False)
 
     return signals
+
+def _process_session(signals, sessions, detrend, standardize, confounds, low_pass, high_pass, t_r):
```

Comment:
I think that what is called a session here is actually a run in BIDS language.

Reply:
Oh, what you mean is that you are processing a set of runs, so rather `_process_runs` maybe

Reply:
I agree that's a more sensible name